packages/@glimmer/syntax/lib/errors/syntax-error.ts

```diff
@@ -0,0 +1,17 @@
+import * as AST from '../types/nodes';
+
+/*
+ * Subclass of `Error` with additional information
+ * about location of incorrect markup.
+ */
+class SyntaxError extends Error {
```

Comment:
Can you double check what this transpiles to? I vaguely recall issues with subclassing Error in IE9 and IE10. It may not matter as much here though since this code is primarily ran in node-land...